### PR TITLE
Bump knmstate to v0.15.1

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: pkg/components/components.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ const (
 	LinuxBridgeCniImageDefault    = "quay.io/kubevirt/cni-default-plugins:v0.8.1"
 	LinuxBridgeMarkerImageDefault = "quay.io/kubevirt/bridge-marker:0.2.0"
 	KubeMacPoolImageDefault       = "quay.io/kubevirt/kubemacpool:v0.8.1"
-	NMStateHandlerImageDefault    = "quay.io/nmstate/kubernetes-nmstate-handler:v0.14.0"
+	NMStateHandlerImageDefault    = "quay.io/nmstate/kubernetes-nmstate-handler:v0.15.1"
```
Comment (phoracek) [inline pkg/components/components.go]: This is weird, I'm pretty sure we had 0.15.0 here, what did I do wrong

Comment (qinqon) [reply pkg/components/components.go]: I did review 0.15.0 knmstate bump wrong :-/

**Comment by phoracek** [comment]
/override pull-e2e-cluster-network-addons-operator-lifecycle-k8s

All passed locally.

**Comment by phoracek** [comment]
/override pull-e2e-cluster-network-addons-operator-workflow-k8s

Passed locally except for the two tests that always fail :/

**Pull request merged**